lib/fsext/unprependfs.go

```diff
@@ -0,0 +1,172 @@
+package fsext
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/spf13/afero"
+)
+
+var _ afero.Lstater = (*UnprependPathFs)(nil)
+
+// UnprependPathFs is a filesystem that wraps another afero.Fs and unprepend a given path from all
```

Comment:
I don't think unprepend is a word :grin: `TrimPrefixFs`? :smile:

Reply:
I was wondering if I should name it PathChangeFs and have it take a function that gets the path and changes it ... But maybe TrimPrefixFs is good for now

Reply:
Hmm `PathChangeFs` seems a good idea, especially since the `normalizedFs` is basically that, but only partially implemented, right?